src/pages/game/index.js

```diff
@@ -0,0 +1,90 @@
+const game = function() {
+  const $root = document.querySelector("#root");
+  const $cardsWrapper = createCardsWrapper();
+  const createMemoryCard = memoryCard.create();
+  const $pointBar = pointBar.create();
+  const $backBtn = backBtn.render("Back");
+  const $layerStart = layerStart.render("Start", "Back");
+
+  const $memoryCardC = createMemoryCard({
+    src: "img/icon-c.png",
+    alt: "imagem do livro de C++",
+    nameClass: "-front"
+  });
+
+  const $memoryCardJS = createMemoryCard({
+    src: "img/icon-js.png",
+    alt: "imagem do livro de Javascript",
+    nameClass: "-front"
+  });
+
+  const $memoryCardJava = createMemoryCard({
+    src: "img/icon-java.png",
+    alt: "imagem do livro de Java",
+    nameClass: "-front"
+  });
+
+  // const $memoryCardCabeca = createMemoryCard({
+  //   src: "img/icon-cabeca.png",
+  //   alt: "imagem do livro de código na cabeça",
+  //   nameClass: "-front"
+  // });
```

Comment:
Ah uma dica aqui, sempre que você for commitar o código limpa os comentários de códigos que não serão mais utilizados, é uma boa prática para evitar atrapalhar o entendimento de quem estiver lendo e também não corre o risco de alguém descomentar e causar um possível bug ou comportamento inesperado.

Reply:
Ok, entendido, obrigado pela dica.